package.json

```diff
@@ -59,7 +59,8 @@
       "rollup@>=4.0.0 <4.22.4": "^4.22.4",
       "vite@>=5.0.0 <5.4.12": "^5.4.12",
       "ws@>=8.0.0 <8.17.1": "^8.18.0",
-      "xml-crypto@3.0.1": "^3.2.1"
+      "xml-crypto@3.0.1": "^3.2.1",
+      "tar-fs@3.0.6": "^3.0.8"
```

Comment:
Can we use range version number? Like
```
"tar-fs@<3.0.8": "^3.0.8"
```

Reply:
will this potentially affect 2.x.x?

Reply:
if you don't want to impact v2, you can also specify it like this:
```
"tar-fs@>=3.0.0 <3.0.8": "^3.0.8"
```

Reply:
updated